### ISSUE TITLE
Parsing dots inside brackets

### ISSUE THREAD

**Issue by dchusovitin** [hapijs/qs#113]
This bug introduced on 6ee5df78 commit

```js
// Example
// parseResult should be equal to data

var qs = require('./lib/index');

var data =  {
	b: {
		'x.y': 'z'
	},
};

// 'b%5Bx.y%5D=z'
var strResult = qs.stringify(data);

// { b: { y: 'z' } }
var parseResult = qs.parse(strResult);
```

**Comment by hueniverse** [comment]
We need to revert this change (go back to the previous default). It will be a major release.

**Comment by nlf** [comment]
will be publishing a new major release with allowDots defaulted to false shortly

**Comment by nlf** [comment]
closed via 31d0f0374ebfdc38d8ac74ef102b10cbbfdc4122
